lib/php/rendition-factory/src/Transformer/Video/FFMpeg/Format/JpegFormat.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Alchemy\RenditionFactory\Transformer\Video\FFMpeg\Format;
+
+use Alchemy\RenditionFactory\DTO\FamilyEnum;
+use FFMpeg\Format\Video\X264;
+
+class JpegFormat implements FormatInterface
+{
+    public static function getAllowedExtensions(): array
+    {
+        return ['jpg', 'jpeg'];
+    }
+
+    public static function getMimeType(): string
+    {
+        return 'image/jpeg';
+    }
+
+    public static function getFormat(): string
+    {
+        return 'image-jpeg';
```

Comment:
Is the prefix `image-` relevant?

Reply:
to be consistant with other formats, including specifics like `image-gif` and `animated-gif`